.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--require rails_helper
```

Comment:
The idea behind the rails_helper is you directly require it in the spec files where it's needed and not globally.  This way if you have tests that don't need anything Rails specific, they don't need to include the rails_helper, and those tests can be run from the command line *really fast*.

That being said I'm not sure I *agree* with that position, and I'm fine with your PR as is, but in that case we may just want to squash the 2 helpers.  I'd love to discuss.

See https://stackoverflow.com/a/24145760/2355944